### PR TITLE
Update the README quickstart to point to the docs quick start

### LINKED ISSUE #431: Update README Quickstart to suggest a `pip install`
### Docs Update Description

Having users need both `Docker` and `Make` just to do a quick demo seems like too high of a barrier. We have the resources in the repo for people to to a direct `pip install` and go from there for a quick demo, then pointing them to the real docs for more of an in-depth setup.

### Additional context

Some people might not want to use docker or make, and our quickstart in the README might discourage those people

## UPDATE:

Given the complexity of creating and maintaining a quick start/tutorial, I think the path forward is to deprecate it and point to the quick start

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
Closes #431 

### Code Changes

* [x] refactor the quickstart in the README to point to the docs quick start

### Steps to Confirm

* [ ] _list any manual steps taken to confirm the changes_

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [x] Documentation Updated
* [x] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created

### Description Of Changes

It didn't seem tenable to try to maintain two separate tutorials/quick starts, so the README version has been deprecated and replaced with a link to our actual docs site.

### CODE REVIEW

File: fidesctl/.fides/fidesctl.toml [hunk 1]
```diff
@@ -4,11 +4,11 @@ database_password = "fidesctl"
 database_host = "fidesctl-db"
 database_port = "5432"
 database_name = "fidesctl"
-
 test_database_name = "fidesctl_test"
 
 [cli]
 server_url = "http://fidesctl:8080"
+analytics_id = "internal"
```
Comment (ThomasLaPiana) [inline fidesctl/.fides/fidesctl.toml:11]: @PSalant726 we _don't_ want to commit this, correct?

**Comment by ThomasLaPiana** [comment]
@conceptualshark I'm making some edits here but honestly it just still doesn't feel right. I'm having doubts that having a quick start in our README is even the right choice here. Do we have any qualms about just pointing them to the fleshed out docs demo?

As it is, this quick start is extremely fragile and not actually super helpful I'd argue. If we focus on making the docs quickstart the canonical quick start we can give users a much better experience, in my opinion

**Comment by conceptualshark** [comment]
@ThomasLaPiana I think pointing to the documentation and covering a bit more about what the project is/some use cases might be a better use of our space in the README. 

If it's gotten to the point where we can't cleanly cover "getting started" here without it getting out of date (or needing supplemental info), I'd rather point to one source we can guarantee Works, vs maintaining that information in two different places.

**Comment by ThomasLaPiana** [comment]
@conceptualshark couldn't have said it better myself. I'll commandeer this PR to just tear it our and point to the proper one we work so hard on

**Comment by ThomasLaPiana** [comment]
discussing in slack with @iamkelllly what possible alternatives might look like

**Comment by ThomasLaPiana** [comment]
this is going to get updated by @conceptualshark at a later date

**Pull request closed without merging**